client/signup/steps/import-url/index.jsx

```diff
@@ -15,34 +17,48 @@ import Button from 'components/button';
 import ButtonGroup from 'components/button-group';
 import StepWrapper from 'signup/step-wrapper';
 import SignupActions from 'lib/signup/actions';
-import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
-//import FormInputValidation from 'components/forms/form-input-validation';
-import { fetchIsSiteImportable, setNuxUrlInputValue } from 'state/importer-nux/actions';
```

Comment:
Where are we calling `fetchIsSiteImportable` instead to make sure it's a supported site?

Reply:
I was planning to add the check in the next step / after the url is submitted and before attempting to show the site-preview.